src/main/java/frc/robot/RobotContainer.java

```diff
@@ -33,15 +34,21 @@ public class RobotContainer {
   private ADIS16470_IMU m_gyro = new ADIS16470_IMU();
   private final DriveSubsystem m_robotDrive = new DriveSubsystem(m_gyro);
   private final Intake intake = new Intake();
-  private final Arm m_arm = new Arm();
+  // private final Arm m_arm = new Arm();
```

Comment:
uncomment this out.  we do want to arm.  I only commented it out to prevent issues that came from  the missing motor controllers on the armless chassis.